src/stories/other.stories.tsx

```diff
@@ -77,3 +78,14 @@ stories.add('Circle Icon', withInfo(
         {text('Text', '1')}
     </CircleIcon>
 ));
+
+stories.add('Keep aspect ratio', withInfo(
+    'A container to keep a given aspect ratio for an image or iframe'
+)(() =>
+    <div>
+        <div style={{ width: text('Width', '40rem'), backgroundColor: '#DDD'}} >
```

Comment:
Don't add knobs that are not props to the component as it's confusing

Reply:
okay, removed it